workshop/migrations/0031_auto_20201222_2350.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.2.13 on 2020-12-22 18:20
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('workshop', '0030_auto_20201222_2349'),
```

Comment:
I think you messed up with the migrations. The file `0030_auto_20201222_2349.py` does not exist. Maybe try to fix it. Everything else looks good in my opinion!

Reply:
made migrations again